src/main/kotlin/org/jitsi/utils/stats/RateTracker.kt

```diff
@@ -0,0 +1,156 @@
+/*
+* Copyright @ 2018 - present 8x8, Inc.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*     http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+package org.jitsi.utils.stats
+
+import org.jitsi.utils.ms
+import java.lang.IllegalArgumentException
+import java.time.Clock
+import java.time.Duration
+
+open class RateTracker @JvmOverloads constructor(
+    /**
+     * The number of buckets.
+     */
+    private val numBuckets: Int,
+    /**
+     * The size of each bucket in milliseconds.
+     */
+    private val bucketSizeMs: Int = 1,
```

Comment:
Is this in ms (as opposed to a `Duration`) for backwards compatibility?  Or perf?

Reply:
Neither. I'm only keeping `RateStatistics` for backward comp. I'll clean that up a bit.